components/main-chef-wrapper/lib/version_constants.go

```diff
@@ -0,0 +1,3 @@
+package lib
+
+const ChefCliVersion = "5.4.2"
```

Comment:
Where is this used? Also this version isn't right

Reply:
@tas50 I wonder why it still shows here, it was removed in later commit.

Reply:
Removing this, it was temporary, i was planning to bump i, like in cli.